### ISSUE TITLE
Please mention this repository in googapis

### ISSUE THREAD

**Issue by mfelsche** [mechiru/google-api-proto#4]
We are happy users of the googapis crate, but wanted to update our prost dependency without luck due to gooapis depending on 0.9.
By accident i somehow found this repository and it seems to be a successor to googapis. It would be totally awesome to have the README of https://github.com/mechiru/googapis to point to this repo/crate.

**Comment by felschr** [comment]
Yeah, it's pretty unclear what happened. I think googapis should link to this project as its successor and be archived ideally with an explanation.
